articles/migrations/guides/extensibility-node8.md

```diff
@@ -116,17 +126,24 @@ to
 
 `var mysql = require(‘mysql’);`
 
+
+or, if the module must be pinned to a specific version:
+
+`var mysql = require(‘mysql@2.0.0’);`
+
 ### Behavioral and syntactic changes
 
-Some of the behavioral and syntactic changes in modules were not forward-compatible with Node.js v8.
+Some of the behavioral and syntactic changes in modules were not forward-compatible with Node 8.
 
 For example, the default encoding of the **crypto** module was changed from **binary** to **utf8**, and the used of **new Buffer()** has been deprecated in favor of **Buffer.from()**.
 
 Please consult Node.js' migration nodes for [v4 to v6](https://github.com/nodejs/node/wiki/Breaking-changes-between-v4-LTS-and-v6-LTS) and [v6 to v8](https://github.com/nodejs/node/wiki/Breaking-changes-between-v6-LTS-and-v8-LTS) for additional information.
 
 ## Important Dates
```

Comment:
This should be further up in the doc, even as part of the `Summary of the migration`.